src/text-field-base.tsx

```diff
@@ -1,8 +1,16 @@
 import * as React from 'react';
```

Comment:
I was looking that password bullets are smaller than specs
![image](https://user-images.githubusercontent.com/6722153/140479828-f47d49a9-aff4-433b-a003-36f6f43e0258.png)
https://www.figma.com/file/e8d3PkXitUJ1Jwgc9Y3mNc/Forms-Component-Specs?node-id=0%3A1

Can we solve this in this PR?
We are using in design side, Lucida Grande font in these bullets

Reply:
please create a new ticket for this